hash/nes.xml

```diff
@@ -66644,7 +66644,7 @@ Also notice that VRAM & WRAM are probably incorrect for some of these sets, at t
 			<feature name="pcb" value="UNL-KS7031" />
 			<feature name="mirroring" value="vertical" />
 			<dataarea name="prg" size="131072">
-				<rom name="dracula ii - noroi no fuuin (fds conversion).prg" size="131072" crc="8b7c2f62" sha1="759801bc77163613a21840591b69a336a8e0a06a" offset="00000" status="baddump" />
+				<rom name="ks128" size="131072" crc="8b7c2f62" sha1="759801bc77163613a21840591b69a336a8e0a06a" offset="00000" status="baddump" />
```

Comment:
This looks like an unintentional mistake. Was it intentional?

Reply:
Sorry, I've got a bad habit of throwing in little corrections without mention them. It was intentional. It's on the DIP: https://forums.nesdev.com/download/file.php?id=18313&sid=284bdcde9aab3ba76d935b19d97e9c17&mode=view. I happened to notice it when I happened to notice the wrong mapper number in kaiser.cpp.

Reply:
Heh, I made another change here. lweaponc is a bootleg port of Lethal Enforcer, unrelated to the NES lweapon game.